zhaquirks/tuya/ts0601_switch.py

```diff
@@ -438,6 +438,63 @@ class TuyaTripleSwitch_GP(TuyaSwitch):
     }
 
 
+class TuyaTripleSwitch_var03(TuyaSwitch):
```

Comment:
Do you want to rename this to something like `TuyaTripleSwitchVar03`?
It would be more conform to https://peps.python.org/pep-0008/#class-names

I know some Tuya quirk classes still use the `_`, but ideally, we should move away from it and follow conventions to use CapCase/CamelCase for class names.

Reply:
Done.
I'm sorry for spreading the wrong format in most of the quirks.

Reply:
All good! It's just a minor thing.